src/main/java/com/worktracker/model/dto/ReportTaskDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.worktracker.model.dto;
+
+import com.worktracker.model.TypeTask;
+
+public class ReportTaskDTO {
+
+    private final Long taskId;
+    private final TypeTask typeTask;
+    private final String taskName;
+    private Double taskHours;
+
+    public ReportTaskDTO(Long taskId, TypeTask typeTask, String taskName, Double taskHours) {
+        this.taskId = taskId;
+        this.typeTask = typeTask;
+        this.taskName = taskName;
+        this.taskHours = taskHours;
+    }
+
+    public Long getTaskId() {
+        return taskId;
+    }
+
+    public TypeTask getTypeTask() {
+        return typeTask;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public void addHours(Double taskHours) {
+        this.taskHours += taskHours;
```

Comment:
If your `taskHours` are null this will throw NPE. Just set default value for taskHours.